pkgs/servers/misc/subsonic/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, jre }:
+
+let version = "5.2.1"; in
+
+stdenv.mkDerivation rec {
+  name = "subsonic-${version}";
+  inherit version;
+
+  src = fetchurl {
+    url = "mirror://sourceforge/subsonic/subsonic-${version}-standalone.tar.gz";
+    sha256 = "523fa8357c961c1ae742a15f0ceaabdd41fcba9137c29d244957922af90ee791";
+  };
+
+  inherit jre;
+
+  builder = ./builder.sh;
```

Comment:
Initially, I created a launcher script for subsonic in the builder script as well. Since I no longer need that script, I've moved the contents of this script into the `unpackPhase` and `installPhase` in `default.nix`.